doc/reference/networking/lwm2m.rst

```diff
@@ -11,8 +11,281 @@ Overview
+| 11 | Error Code        | R   | Multiple | Optional  | Integer |
+|    |                   |     |          |           | 0-8     |
++----+-------------------+-----+----------+-----------+---------+
+| 12 | Reset Error       | E   | Single   | Optional  |         |
++----+-------------------+-----+----------+-----------+---------+
+| 13 | Current Time      | RW  | Single   | Optional  | Time    |
++----+-------------------+-----+----------+-----------+---------+
+| 14 | UTC Offset        | RW  | Single   | Optional  | String  |
++----+-------------------+-----+----------+-----------+---------+
+| 15 | Timezone          | RW  | Single   | Optional  | String  |
++----+-------------------+-----+----------+-----------+---------+
+| 16 | Supported Binding | R   | Single   | Mandatory | String  |
++----+-------------------+-----+----------+-----------+---------+
+| 17 | Device Type       | R   | Single   | Optional  | String  |
++----+-------------------+-----+----------+-----------+---------+
+| 18 | Hardware Version  | R   | Single   | Optional  | String  |
++----+-------------------+-----+----------+-----------+---------+
+| 19 | Software Version  | R   | Single   | Optional  | String  |
++----+-------------------+-----+----------+-----------+---------+
+| 20 | Battery Status    | R   | Single   | Optional  | Integer |
+|    |                   |     |          |           | 0-6     |
++----+-------------------+-----+----------+-----------+---------+
+| 21 | Memory Total (Kb) | R   | Single   | Optional  | Integer |
++----+-------------------+-----+----------+-----------+---------+
+| 22 | ExtDevInfo        | R   | Multiple | Optional  | ObjLink |
++----+-------------------+-----+----------+-----------+---------+
+
+The server could query the `Device Manufacturer` for `Device` object instance 0
+(the default and only instance) by sending a `READ 3/0/0` operation to the
+client.
+
+The full list of registered objects and resource IDs can be
+found in the [LwM2M registry](http://www.openmobilealliance.org/wp/OMNA/LwM2M/LwM2MRegistry.html).
+
+Zephyr's LwM2M library lives in the `subsys/net/lib/lwm2m directory`, with a
+client sample in `samples/net/lwm2m_client`.  For more information about the
+provided sample see: ref:`lwm2m-client-sample`  The sample can be configured
+to use normal unsecure network sockets or sockets secured via DTLS.
+
+The Zephyr LwM2M library implements the following items:
+
+* engine to process networking events and core functions
+* RD client which performs BOOTSTRAP and REGISTRATION functions
+* TLV, JSON and plain text formatting functions
+* LwM2M Technical Specification Enabler objects such as Security, Server,
+  Device, Firmware Update, etc.
+* Extended IPSO objects such as Light Control, Temperature Sensor and Timer
+
+The library currently implements up to [LwM2M specification 1.0.2](http://openmobilealliance.org/release/LightweightM2M/V1_0_2-20180209-A/OMA-TS-LightweightM2M-V1_0_2-20180209-A.pdf).
```

Comment:
This link format (using ``[link name](http://xxx)``  is for markdown and doesn't work for ReST.  (Jukkar's suggestion is correct.)